locust/contrib/fasthttp.py

```diff
@@ -378,7 +378,7 @@ def rest(
         with self.client.request(method, url, catch_response=True, headers=headers, **kwargs) as r:
             resp = cast(RestResponseContextManager, r)
             resp.js = None  # type: ignore
-            if resp.text is None:
+            if resp.content is None:
                 resp.failure(str(resp.error))
             elif resp.text:
```

Comment:
does this really solve your issue? wont it just crash on this line instead?

Reply:
No it doesn't crash, Since in text property I am trying to avoid encoding the string. The issue comes from there where we try to convert to test using `None` encoding